### DISCUSSION TITLE
Cannot read property 'levels' of undefined

### DISCUSSION THREAD

**Discussion by brieucdlf** [cssbattle/forum#55] [Bugs reports]
Hey guys !

Added a friend on my friend list and since then, I got this error and friend leaderboard not loading:
![Screenshot from 2021-05-02 23-50-16](https://user-images.githubusercontent.com/2735645/116828781-314ac480-aba1-11eb-8865-d1c8f03f5db0.png)

**Comment by chinchang** [comment]
Hi @brieucdlf Does this error show when you are on the friends leaderboard? Or even on global leaderboard?

**Reply by brieucdlf** [reply]
On the home page as you can see here :)

![Screenshot from 2021-05-06 17-38-32](https://user-images.githubusercontent.com/2735645/117326415-e9be8400-ae91-11eb-99f7-80f9eb313553.png)

**Comment by brieucdlf** [comment]
@chinchang Any news on this, still have the bug :)
